Sources/CMakeLists.txt

```diff
@@ -5,66 +5,90 @@
 project(Alusus)
 cmake_minimum_required(VERSION 3.1)
 
+if(MINGW)
+  set(InstallLibDir "${CMAKE_INSTALL_PREFIX}/Bin")
+  include_directories("${CMAKE_CURRENT_SOURCE_DIR}/Win32Helpers")
+else()
+  set(InstallLibDir "${CMAKE_INSTALL_PREFIX}/Lib")
+endif()
+link_directories(${InstallLibDir})
+
 # Read version info from version file.
-execute_process(COMMAND python3 version_info.py
-                WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../Tools/BuildSrc
-                OUTPUT_VARIABLE VersionInfo)
-string(REGEX MATCH "VERSION: ([0-9]+\\.[0-9]+\\.[0-9]+)" _dummy_ ${VersionInfo})
-set(AlususVersion ${CMAKE_MATCH_1})
-string(REGEX MATCH "^[0-9]+\\.[0-9]+" AlususSoVersion ${AlususVersion})
-string(REGEX MATCH "REVISION: ([A-Za-z0-9]+)" _dummy_ ${VersionInfo})
-set(AlususRevision ${CMAKE_MATCH_1})
-if (NOT ${AlususRevision} STREQUAL "")
-  set(AlususRevision "-${AlususRevision}")
+
+if(PYTHON_PREFIX)
+  execute_process(COMMAND ${PYTHON_EXECUTABLE} version_info.py ${PYTHON_PREFIX}
+                  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/../Tools/BuildSrc
+                  OUTPUT_VARIABLE VersionInfo)
```

Comment:
<p dir=rtl>
ماذا يفعل  PYTHON_PREFIX؟
</dir>

Reply:
<p dir=rtl>
وماذا يحصل إن لم يكن معرّفا؟
</p>

Reply:
<p dir='rtl' align='right'>جملة تجربة</p>

Reply:
The `PYTHON_PREFIX` variable is needed since I moved to downloading Python packages in the builds folder, rather than the user's home folder like what it was previously. The `PYTHON_PREFIX` contains all downloaded Python packages.